daemon/logger/loggerutils/rotatefilewriter.go

```diff
@@ -33,45 +44,70 @@ func NewRotateFileWriter(logPath string, capacity int64, maxFiles int) (*RotateF
 	}
 
 	return &RotateFileWriter{
-		f:            log,
-		capacity:     capacity,
-		currentSize:  size,
-		maxFiles:     maxFiles,
-		notifyRotate: pubsub.NewPublisher(0, 1),
+		f:             log,
+		buf:           bytes.NewBuffer(nil),
+		capacity:      capacity,
+		currentSize:   size,
+		lastTimestamp: defaultTimestamp,
+		maxFiles:      maxFiles,
+		notifyRotate:  pubsub.NewPublisher(0, 1),
 	}, nil
 }
 
-//WriteLog write log message to File
-func (w *RotateFileWriter) Write(message []byte) (int, error) {
-	w.mu.Lock()
+//WriteLog write log logEntry to File
+func (w *RotateFileWriter) Write(logEntry *jsonlog.JSONLogs) error {
```

Comment:
@cpuguy83 
It's really nice, I will implement it in a new PR, Thanks for your guidance!  : )